subsys/net/lib/nrf_cloud/Kconfig

```diff
@@ -22,6 +22,7 @@ config NRF_CLOUD_STATIC_IPV4_ADDR
 
 config NRF_CLOUD_PROVISION_CERTIFICATES
 	bool "nRF Cloud library provision of certificate"
+	select NRF_INBUILT_KEY
```

Comment:
We should not do this, asset_tracker for instance, has a config for `qemu_x86` which is useful for testing. Therefore, there should be no direct dependencies to nRF91 in the project.

Reply:
How do we hide this option from a platform that does not support it then? Like nrf52 ?

Reply:
`select NRF_INBUILT_KEY if BSD_LIBRARY` seems like a valid syntax, probably that would be an alternative.

Reply:
Updated to the proposed statement `if BSD_LIBRARY`.